### PR TITLE
Last methods for nosql_execution_Store.go covered

### PR DESCRIPTION
**Pull request by agautam478**
**What changed?**
Happy paths for all methods in the file are done. Will be working on covering the missed lines from now onwards.

**Why?**
increase code coverage.

**How did you test it?**
unit tests

**Potential risks**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: common/persistence/nosql/nosql_execution_store_test.go [hunk 1]
```diff
@@ -1067,6 +1067,182 @@ func TestNosqlExecutionStore(t *testing.T) {
...
+					Return(errors.New("database error"))
+			},
+			expectedError: &types.InternalServiceError{Message: "database error"},
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			controller := gomock.NewController(t)
+			defer controller.Finish()
```
Comment (taylanisikdemir) [inline common/persistence/nosql/nosql_execution_store_test.go:1109]: not needed

File: common/persistence/nosql/nosql_execution_store_test.go [hunk 2]
```diff
@@ -1067,6 +1067,182 @@ func TestNosqlExecutionStore(t *testing.T) {
...
```
Comment (taylanisikdemir) [inline common/persistence/nosql/nosql_execution_store_test.go:1172]: same

File: common/persistence/nosql/nosql_execution_store_test.go [hunk 3]
```diff
@@ -1067,6 +1067,182 @@ func TestNosqlExecutionStore(t *testing.T) {
...
+					InsertReplicationTask(ctx, gomock.Any(), nosqlplugin.ShardCondition{ShardID: shardID, RangeID: 123}).
+					Return(nil)
+			},
+			expectedError: nil,
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			controller := gomock.NewController(t)
+			defer controller.Finish()
```
Comment (taylanisikdemir) [inline common/persistence/nosql/nosql_execution_store_test.go:1225]: same

**Review by taylanisikdemir — APPROVED** [review]

File: common/persistence/nosql/nosql_execution_store_test.go (hunk 2 above)
Comment (agautam478) [reply common/persistence/nosql/nosql_execution_store_test.go:1172]: removing

File: common/persistence/nosql/nosql_execution_store_test.go (hunk 3 above)
Comment (agautam478) [reply common/persistence/nosql/nosql_execution_store_test.go:1225]: removing

File: common/persistence/nosql/nosql_execution_store_test.go (hunk 1 above)
Comment (agautam478) [reply common/persistence/nosql/nosql_execution_store_test.go:1109]: removing

**Pull request merged**